infrastructure/general/setup-full.md

```diff
@@ -0,0 +1,366 @@
+---
+title: Set up a Full Node
+description: Instructions on how to set up a full node in accordance to your platform and the differences between different node types.
+---
+
+If you're building dApps or products on a Substrate-based chain like Polkadot, Kusama, or a custom
+Substrate implementation, you want the ability to run a node-as-a-back-end. After all, relying on
+your infrastructure is always better than a third-party-hosted one in this brave new decentralized
+world.
+
+This guide will show you how to connect to [Polkadot](https://polkadot.network/), but the
+same process applies to any other [Substrate](https://substrate.io)-based chain.
+
+<!-- TODO - maybe we keep this as a "intro" page for the infrastructure section, and make this page more focused post-merge? -->
+
+### Types of Nodes
```

Comment:
I think it would be nice to have sub-headers like:
#### Full Node
#### Light Node
and so on..

Reply:
> I think it would be nice to have sub-headers like:
> 
> #### Full Node
> #### Light Node
> and so on..

This would be better, as now it seems like a massive amount of text